Jenkinsfile

```diff
@@ -1068,35 +1076,138 @@ pipeline {
           steps {
             sh 'python examples/nlp/token_classification/token_classification_evaluate.py \
             model.dataset.data_dir=/home/TestData/nlp/ner/ \
+            model.dataset.use_cache=false \
             pretrained_model=/home/TestData/nlp/pretrained_models/NER_Model_with_BERT_base_uncased.nemo'
           }
         }
         stage('Evaluation script for Punctuation') {
           steps {
             sh 'python examples/nlp/token_classification/punctuation_capitalization_evaluate.py \
-            model.dataset.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
+            model.train_ds.ds_item=/home/TestData/nlp/token_classification_punctuation/ \
+            model.validation_ds.ds_item=/home/TestData/nlp/token_classification_punctuation/ \
+            model.test_ds.ds_item=/home/TestData/nlp/token_classification_punctuation/ \
+            +model.train_ds.use_cache=false \
+            +model.validation_ds.use_cache=false \
+            +model.test_ds.use_cache=false \
             pretrained_model=/home/TestData/nlp/pretrained_models/Punctuation_Capitalization_with_DistilBERT_base_uncased.nemo'
           }
         }
-        stage('L2: Punctuation & Capitalization, 2GPUs with DistilBERT') {
+        stage('L2: Punctuation & Capitalization, 2GPUs with DistilBERT, Fine-tuning on different data') {
           steps {
             sh 'cd examples/nlp/token_classification && \
+            mkdir -p tmp_data && \
+            cp /home/TestData/nlp/token_classification_punctuation/*.txt tmp_data/ && \
             python punctuation_capitalization_train.py \
-            model.dataset.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
-            model.language_model.pretrained_model_name=distilbert-base-uncased \
-            model.dataset.use_cache=false \
-            trainer.gpus=[0,1] \
-            trainer.accelerator=ddp \
-            trainer.max_epochs=1 \
-            +exp_manager.explicit_log_dir=/home/TestData/nlp/token_classification_punctuation/output && \
+              model.train_ds.ds_item=tmp_data \
+              model.validation_ds.ds_item=tmp_data \
+              model.test_ds.ds_item=tmp_data \
+              model.language_model.pretrained_model_name=distilbert-base-uncased \
+              +model.train_ds.use_cache=false \
+              +model.validation_ds.use_cache=false \
+              +model.test_ds.use_cache=false \
+              trainer.gpus=[0,1] \
+              trainer.accelerator=ddp \
+              trainer.max_epochs=1 \
+              +exp_manager.explicit_log_dir=/home/TestData/nlp/token_classification_punctuation/output \
+              +do_testing=true && \
```

Comment:
with this new `do_testing` flag, `punctuation_capitalization_evaluate.py` can be deleted, maybe move its inference part to the `punctuate_capitalize_infer.py`